### ISSUE TITLE
Reset is not functioning

### ISSUE THREAD

**Issue by ahmedgroup1989** [stape-io/mixpanel-tag#8]
Hello
I installed the template and I fire the reset by passing these data and choosing type "reset" from tag and tag fires but each time it links any event to a certain distinct id (58496116) even if i logout , so now i can only use server side for loggged in user but for guests it will be linked to that distinct id

![image](https://github.com/user-attachments/assets/4c50e45c-1634-47e2-b671-fb53fc5d6a67)

**Comment by Bukashk0zzz** [comment]
Hi,

If you use incognito mode of the browser do you still get same id?
Look like issue connected to some clean storage issue.

**Comment by ahmedgroup1989** [comment]
Yes the same for any user from any other browser as well and incognito

**Comment by gvidoou** [comment]
@ahmedgroup1989 
Checked the flow, looks like mixpanel web tag resets cookies (mp_<project_id>_mixpanel) with distinct_id and device_id but when you refresh page it maps your new distinct_id to the same user on mixpanel.  Stape server tag works the same, reset just remove distinct_id and device_id cookie and next time will generate new cookies.

This Distinct ID is probably yours, when you login to mixpanel, you can find it in cookies "mp_distinct_id"
![image](https://github.com/user-attachments/assets/2bb7a707-bbed-4d35-8988-5b1720d06b83)
